### ISSUE TITLE
Feature Request: Display Total and Selected API Count in Postman Runner

### ISSUE THREAD

**Issue by loutsiros** [a85/POSTMan-Chrome-Extension#13116]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

When using the Postman Runner, there are options to "Deselect All APIs," "Select All APIs," and "Reset," there is no indication of the total number of APIs or the count of selected APIs displayed in the application. 

<img width="954" alt="66" src="https://github.com/user-attachments/assets/89e0acee-3976-4c2a-93d8-b4a1a14fa59e">

### Describe the solution you'd like

It would be incredibly helpful if these counts were shown at the top of the Postman Runner, making it easier to manage and review API selections

**Comment by atifrnkhan8090** [comment]
Thanks, @loutsiros , for considering this and taking it as a feature request.

**Comment by asif-jamal** [comment]
I agree, this feature would be really useful. I also think there should be a count showing the total APIs and the selected APIs. I support this ide great suggestion and thanks for bringing it up!
Asif Jamal,
Persistent Systems | Lead Software Engineer
